src/asmcnc/comms/logging/log_exporter.py

```diff
@@ -0,0 +1,113 @@
+from datetime import datetime
+import os
+import paramiko
```

Comment:
put paramiko import into try-except. we install it with cryptography on the pis, but will be good for ensuring dev systems can run without freaking out :)